### PR TITLE
Add Docker Desktop 4.24.1 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
### Proposed changes

Docker desktop 4.21.1 release notes

### CODE REVIEW

File: content/desktop/release-notes.md [hunk 1]
```diff
@@ -24,11 +24,23 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{{< release-date date="2023-10-04" >}}
+
+{{< desktop-install win=true version="4.24.1" build_path="/" >}}
+
+### Bug fixes and enhancements
+
+#### For Windows
+
+- Fixed a bug on Windows where the Dashboard wouldn't show container logs correctly. Fixes [docker/for-win#13714](https://github.com/docker/for-win/issues/13714).
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:37]:
```suggestion
- Fixed a bug on Docker Desktop for Windows where the Docker Dashboard wouldn't display container logs correctly. Fixes [docker/for-win#13714](https://github.com/docker/for-win/issues/13714).
```

**Review by aevesdocker — DISMISSED** [review]
Merci @lorenrh!

**Comment by lorenrh** [comment]
Applied the suggestion, and added a Known issues section for mac.

File: content/desktop/release-notes.md [hunk 2]
```diff
@@ -87,7 +99,11 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 - Fixed a bug that caused Docker Desktop fail to start on Windows hosts. Fixes [docker/for-win#13662](https://github.com/docker/for-win/issues/13662).
 - Modified the Docker Desktop resource saver feature to skip reducing kernel memory on WSL when no containers are running, as this was causing timeouts in some cases. Instead, users are encouraged to enable "autoMemoryReclaim" on WSL directly via the .wslconfig file (available since WSL 1.3.10).
 
+### Known Issues
+
+#### For Mac
 
+- Creating a container with the port 53 fails with the error address `already in use`. As a workaround, deactivate network acceleration by adding `"kernelForUDP": false`, in the settings file located at `~/Library/Group Containers/group.com.docker/settings.json`.
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:106]:
```suggestion
- Creating a container with the port 53 fails with the error address `already in use`. As a workaround, deactivate network acceleration by adding `"kernelForUDP": false`, in the `settings.json` file located at `~/Library/Group Containers/group.com.docker/settings.json`.
```

**Review by aevesdocker — APPROVED** [review]

File: content/desktop/release-notes.md [hunk 3]
```diff
@@ -87,7 +99,11 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 - Fixed a bug that caused Docker Desktop fail to start on Windows hosts. Fixes [docker/for-win#13662](https://github.com/docker/for-win/issues/13662).
 - Modified the Docker Desktop resource saver feature to skip reducing kernel memory on WSL when no containers are running, as this was causing timeouts in some cases. Instead, users are encouraged to enable "autoMemoryReclaim" on WSL directly via the .wslconfig file (available since WSL 1.3.10).
 
+### Known Issues
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:102]:
```suggestion
### Known issues
```

**Pull request merged**